govcd/api_vcd_test.go

```diff
@@ -163,6 +163,7 @@ type TestConfig struct {
 		ExternalNetworkPortGroup     string `yaml:"externalNetworkPortGroup,omitempty"`
 		ExternalNetworkPortGroupType string `yaml:"externalNetworkPortGroupType,omitempty"`
 		VimServer                    string `yaml:"vimServer,omitempty"`
+		LdapServer                   string `yaml:"ldap_server,omitempty"`
```

Comment:
The ugly part of GitHub - I can't comment on unaltered code, but could you remove `Misc.LdapContainer` field from here as we don't really need it anymore.

Reply:
Done